presentation/src/main/java/org/sagebionetworks/research/presentation/inject/StepViewModule.java

```diff
@@ -63,7 +64,7 @@
 
     public interface StepViewFactory {
         @Nullable
-        StepView apply(Step step);
+        StepView apply(Step step, DrawableMapper mapper);
```

Comment:
Do steps have different DrawableMappers? I would've thought the StepViewFactory would use one DrawableMapper across steps

Reply:
Yeah they are all using the same one, I'll fix that.